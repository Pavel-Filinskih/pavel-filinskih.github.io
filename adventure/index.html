<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Adventure</title>
    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700&amp;subset=cyrillic" rel="stylesheet">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/style.css">
  </head>
  <body>
    <div class="wrapper-bg-header">
      <div class="wrapper-header-center">
        <header class="header-nav">
          <a class="logo" href="#"><img src="svg/logo.svg" width="41" height="56"></a>

          <h1 class="header-title">adventure <span>оставь свой след, где его еще нет...</span></h1>
        </header>
      </div>
    </div>

    <div class="wrapper-filters-center">
      <div class="filters">
        <ul class="filters-list">
          <li class="filters-item"><a class="filter-link active-filter" href="#">Все страны</a></li>
          <li class="filters-item"><a class="filter-link" href="#">культура</a></li>
          <li class="filters-item"><a class="filter-link" href="#">экстрим</a></li>
          <li class="filters-item"><a class="filter-link" href="#">атмосфера</a></li>
          <li class="filters-item"><a class="filter-link" href="#">Люди</a></li>
        </ul>
      </div>
    </div>

    <div class="wrapper-catalog-center">

        <div class="catalog-countries">
        <div class="column-first">

          <div class="cuba-slider">
      			<div class="cuba-slider__slides">

              <div class="cuba-slider__control">
                <button class="cuba-slider__btn-prev"></button>
                <button class="cuba-slider__btn-next"></button>
              </div>

      				<div class="cuba-slide  cuba-slide--show">
      					<div class="cuba-slide__description">
      						<img class="cuba-image" src="img/cuba-intro.jpg" tabindex="0" width="300" height="200" alt="cuba">
                  <b class="cuba-slide__title  cuba-slide__title--center">Cuba</b>
                </div>
      				</div>
              <div class="cuba-slide">
      					<div class="cuba-slide__description">
      						<b class="cuba-slide__title  cuba-slide__title--small">Cuba</b>
                  <p class="cuba-slide__text">
                    Хэмингуэй, став весьма знаменитым,
                    естественно, мог выбрать любое место
                    в мире. Почему же он 22 года прожил
                    в Гаване? Потому что это свобода!
                  </p>
      					</div>
      				</div>
      				<div class="cuba-slide">
      					<div class="cuba-slide__description  icon-zoom">
                  <img class="cuba-image" src="img/cuba-img.jpg" width="300" height="200" alt="cuba">
      						<b class="cuba-slide__title  cuba-slide__title--small">Cuba</b>
      					</div>
      				</div>
      				<div class="cuba-slide">
      					<div class="cuba-slide__description  icon-play">
                  <img class="cuba-image" src="img/cuba-women.jpg" width="300" height="200" alt="cuba">
                  <b class="cuba-slide__title  cuba-slide__title--small">Cuba</b>
      					</div>
      				</div>
      			</div>
        	</div>

          <div class="england">
            <div class="england-preview">
              <h2 class="england-title">England</h2>
              <img class="england-image" src="img/england-intro.jpg" width="300" height="200" alt="england">
            </div>
            <div class="england-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>

          <div class="switzerland">
            <div class="switzerland-preview">
              <h2 class="switzerland-title">Switzerland</h2>
              <img class="switzerland-image" src="img/switzerland-intro.jpg" width="300" height="200" alt="switzerland">
            </div>
            <div class="switzerland-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>

          <div class="australia">
            <div class="australia-preview">
              <h2 class="australia-title">Australia</h2>
              <img class="australia-image" src="img/australia-intro.jpg" width="300" height="200" alt="australia">
            </div>
            <div class="australia-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>
        </div>

        <div class="column-second">
          <div class="russia">
            <div class="russia-preview">
              <h2 class="russia-title">Russia</h2>
              <img class="russia-image" src="img/russia-intro.jpg" width="300" height="200" alt="Russia">
            </div>
            <div class="russia-control">
              <button class="btn-prev"></button>
              <button class="btn-next"></button>
            </div>
          </div>

          <div class="norway">
            <div class="norway-preview">
              <h2 class="norway-title">Norway</h2>
              <img class="norway-image" src="img/norway-intro.jpg" width="300" height="200" alt="norway">
            </div>
            <div class="norway-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>

          <div class="china">
            <div class="china-preview">
              <h2 class="china-title">China</h2>
              <img class="china-image" src="img/china-intro.jpg" width="300" height="200" alt="china">
            </div>
            <div class="china-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>

          <div class="india">
            <div class="india-preview">
              <h2 class="india-title">India</h2>
              <img class="india-image" src="img/india-intro.jpg" width="300" height="200" alt="india">
            </div>
            <div class="india-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>
        </div>

      <div class="column-latina">
        <div class="latina">
          <div class="latina-preview">
            <h2 class="latina-title">Latin America</h2>
            <img class="latina-image" src="img/latina-intro.jpg" width="620" height="420" alt="latina">
            <ul class="latina-list">
              <ol>1. Honduras</ol>
              <ol>2. Columbia</ol>
              <ol>3. Panama</ol>
              <ol>4. Brazil</ol>
              <ol>5. Peru</ol>
            </ul>
          </div>
        </div>
      </div>

        <div class="column-third">
          <div class="finland">
            <div class="finland-preview">
              <h2 class="finland-title">Finland</h2>
              <img class="finland-image" src="img/finland-intro.jpg" width="300" height="200" alt="finland">
            </div>
            <div class="finland-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>
          <div class="germany">
            <div class="germany-preview">
              <h2 class="germany-title">Germany</h2>
              <img class="germany-image" src="img/germany-intro.jpg" width="300" height="200" alt="germany">
            </div>
            <div class="germany-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>
        </div>

        <div class="column-fourth">
          <div class="zealand">
            <div class="zealand-preview">
              <h2 class="zealand-title">New Zealand</h2>
              <img class="zealand-image" src="img/zealand-intro.jpg" width="300" height="200" alt="zealand">
            </div>
            <div class="zealand-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>
          <div class="japan">
            <div class="japan-preview">
              <h2 class="japan-title">Japan</h2>
              <img class="japan-image" src="img/japan-intro.jpg" width="300" height="200" alt="japan">
            </div>
            <div class="japan-control">
              <button class="control-btn btn-prev"></button>
              <button class="control-btn btn-next"></button>
            </div>
          </div>
        </div>

        </div>
      </div>
    </div>

    <div class="wrapper-bg-footer">
      <div class="wrapper-footer-center">
        <footer class="footer">
          <span class="footer-copyring">&copy; 2013 - 2015</span>
          <p class="footer-maker">Здесь наследили <span>соискатели:)</span></p>
        </footer>
      </div>
    </div>

    <div class="gallery-overlay overlay-show">
      <span class="gallery-overlay-close"></span>
      <div class="gallery-overlay-preview">
        <img src="" class="gallery-overlay-image" alt="cuba">
      </div>
    </div>

    <script src="js/js-slider.js"></script>
    <script src="js/overlay-gallery.js"></script>
  </body>
</html>
