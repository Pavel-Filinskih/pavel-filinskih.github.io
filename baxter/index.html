<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Главная страница</title>
		<link href="css/reset.css" rel="stylesheet">
		<link href="css/style.css" rel="stylesheet">
	</head>
	<body>

		<div class="main-wrapper">
			<header class="main-header">
				<a class="main-header__link" href="index.html">
					<img class="logo" src="img/logo.png" width="252" alt="логотип"/>
				</a>

				<a class="main-header__faq" href="fag.html"></a>

				<ul class="navigation">
					<li class="navigation__item"><a class="navigation__link navigation__link--active" href="index.html">HOME</a></li>
					<li class="navigation__item"><a class="navigation__link" href="about.html">ABOUT</a></li>
					<li class="navigation__item"><a class="navigation__link" href="services.html">SERVICES</a></li>
					<li class="navigation__item"><a class="navigation__link" href="partners.html">PARTNERS</a></li>
					<li class="navigation__item"><a class="navigation__link" href="solutions.html">SOLUTIONS</a></li>
				</ul>
			</header>

			<div class="people">
				<h1 class="people__title">Fusce a sapien ac justo</h1>
				<p class="people__sub-title">Phasellus at elit velit, ac</p>
			</div>

			<div class="services">
				<div class="support">
					<h2 class="support__title">Dedicated 24/7 Support</h2>
					<p class="support__description">
						Pellentesque tellus tortor, lobortis
						eget venenatis non, aliquet a massa.
						Mauris aliquam ultrices lectus, quis
					</p>
					<a class="support__link" href="readmo.html">Read More</a>
				</div>

				<div class="assured">
					<h2 class="assured__title">Quality Assured Services</h2>
					<p class="assured__description">
						Mauris euismod enim aliquet orci
						cursus aliquet. Nam lectus magna,
						varius nec aliquet in, pharetra eu felis.
					</p>
					<a class="assured__link" href="readmo.html">Read More</a>
				</div>

				<div class="focus">
					<h2 class="focus__title">Our Focus</h2>
					<p class="focus__description">
						Etiam ut ligula orci. Praesent male-suada leo eu magna ultricies male-suada.
						Cras et velit a mi bibendum
					</p>
					<a class="focus__link" href="readmo.html">Read More</a>
				</div>
			</div>

			<div class="welcome">
				<div class="site">
					<h2 class="site__title">Welcome to our site!</h2>
					<p class="site__description">
						<span>Sed a sollicitudin risus. Proin neque velit, rutrum et semper et, dignissim feugiat lacus.
						Donec ornare porta dictum. Quisque ornare, libero ac viverra.</span>
						Magna blandit magna, quis	sollicitudin sapien tortor et ante. Nam ante ipsum, cursus ullamcorper volutpat ornare,
						volutpat ac arcu. Phasellus suscipit, leo sit amet scelerisque porta, neque enim malesuada
						neque, non bibendum magna mauris ullamcorper mauris. Curabitur enim felis, molestie fermentum
						ornare id, pellentesque et massa. Nullam vestibulum mi non urna semper vestibulum.
					</p>
					<a class="site__link" href="readmo.html">Read More</a>
				</div>

				<div class="request">
					<h2 class="request__title">Request a Quote</h2>
					<form class="request-form" name="form" action="action.php">
						<input class="request-form__input" type="text" name="fName" placeholder="First Name">
						<input class="request-form__input" type="text" name="sName" placeholder="Lass Name">
						<input class="request-form__input" type="text" name="email" placeholder="Email Address">
						<button class="request-form__button" type="button">send</button>
					</form>
				</div>
			</div>

			<div class="baxter">
				<div class="planning">
					<h3 class="planning__title">planning</h3>
					<img class="planning__pict" src="img/people_1.jpg" alt="картинка">

					<p class="planning__description">
						nteger luctus risus at lectus vestibulum interdum vitae ac lectus.
						Sed in felis neque. Sed facilisis metus non ligula commodo facilisis ut non tortor.
						Morbi turpis nunc, posuere vitae convallis ornare, eleifend pretium dolor.
						Nam fringilla orci at justo viverra fermentum. Nam fringilla aliquam nibh sed semper.
						Proin justo lectus, semper vitae commodo quis, vestibulum eget sem.
						Sed in felis libero, eget feugiat arcu.
						Praesent id augue pellentesque lorem consectetur dignissim ac in elit.
						Aenean commodo rutrum mauris et viverra.
						Nullam massa risus, aliquet ac sodales non, fermentum et dui.
						Proin nulla dolor, accumsan elementum vestibulum vitae, pellentesque vitae velit.
					</p>
					<a class="planning__link" href="readmore.html">Read More</a>
				</div>

				<div class="team">
					<h3 class="team__title">our team</h3>
					<img class="team__pict" src="img/people_2.jpg" alt="картинка">

					<p class="team__description">
						nteger luctus risus at lectus vestibulum interdum vitae ac lectus.
						Sed in felis neque. Sed facilisis metus non ligula commodo facilisis ut non tortor.
						Morbi turpis nunc, posuere vitae convallis ornare, eleifend pretium dolor.
						Nam fringilla orci at justo viverra fermentum. Nam fringilla aliquam nibh sed semper.
						Proin justo lectus, semper vitae commodo quis, vestibulum eget sem. Sed in felis libero, eget feugiat arcu.
						Praesent id augue pellentesque lorem consectetur dignissim ac in elit.
						Aenean commodo rutrum mauris et viverra. Nullam massa risus, aliquet ac sodales non, fermentum et dui.
						Proin nulla dolor, accumsan elementum vestibulum vitae, pellentesque vitae velit.
					</p>
					<a class="team__link" href="readmore.html">Read More</a>
				</div>

				<div class="news">
					<h3 class="news__title">Baxter Blog<span>latest posts</span></h3>

					<div class="new-preview">
						<img class="new-preview__pict" src="img/people_3.jpg" alt="картинка">
						<h4 class="new-preview__title">Sed a sollicitudin risus</h4>
						<p class="new-preview__description">Ullamcorper, elit quis adipiscing egestas, massa leo tincidunt enim, in vehicula nisl lacus nec tortor.</p>
						<a class="new-preview__link" href="readmore.html">read more</a>
					</div>

					<div class="new-preview">
						<img class="new-preview__pict" src="img/people_4.jpg" alt="картинка">
						<h4 class="new-preview__title">Sed a sollicitudin risus</h4>
						<p class="new-preview__description">Ullamcorper, elit quis adipiscing egestas, massa leo tincidunt enim, in vehicula nisl lacus nec tortor.</p>
						<a class="new-preview__link" href="readmore.html">read more</a>
					</div>

					<div class="new-preview">
						<img class="new-preview__pict" src="img/people_5.jpg" alt="картинка">
						<h4 class="new-preview__title">Sed a sollicitudin risus</h4>
						<p class="new-preview__description">Ullamcorper, elit quis adipiscing egestas, massa leo tincidunt enim, in vehicula nisl lacus nec tortor.</p>
						<a class="new-preview__link" href="readmore.html">read more</a>
					</div>
				</div>
			</div>

			<div class="profits">
				<div class="about">
					<h2 class="about__title">About Baxter</h2>

					<p class="about__description">Donec ut tempor massa. Donec faucibus purus in ligula dictum quis consequat elit posuere. In consequat feugiat aliquam. Praesent tempor risus eget ante adipiscing non mollis enim commodo. Vivamus malesuada purus eget nisl hendrerit porttitor. Vestibulum a nibh elit. Phasellus justo nulla, auctor non tempus ut, dapibus sit amet sapien. Fusce turpis libero, rutrum nec fermentum quis, consectetur vel</p>
					<a class="about__link" href="readmore.html">Quisque vestibulum porttitor lorem</a>
				</div>

				<div class="partners">
					<h2 class="partners__title">Our Partners</h2>

					<ul class="partner">
						<li class="partner__item">Donec ut tempor massa. Donec faucibus puru</li>
						<li class="partner__item">s in ligula dictum quis consequat elit posuere. In</li>
						<li class="partner__item">consequat feugiat aliquam. Praesent tempor</li>
						<li class="partner__item">risus eget ante adipiscing non mollis enim</li>
						<li class="partner__item">commodo. Vivamus malesuada purus eget nisl</li>
					</ul>

					<ul class="partner">
						<li class="partner__item">Donec ut tempor massa. Donec faucibus puru</li>
						<li class="partner__item">s in ligula dictum quis consequat elit posuere. In</li>
						<li class="partner__item">consequat feugiat aliquam. Praesent tempor</li>
						<li class="partner__item">risus eget ante adipiscing non mollis enim</li>
						<li class="partner__item">commodo. Vivamus malesuada purus eget nisl</li>
					</ul>
				</div>
			</div>

			<div class="footer">
				<p class="footer__copyring"><a href="index.html">Baxter © 2017&nbsp;|&nbsp;Privacy Policy</a></p>

				<ul class="footer-nav">
					<li class="footer-nav__item"><a class="footer-nav__link footer-nav__link--active" href="index.html">HOME</a></li>
					<li class="footer-nav__item"><a class="footer-nav__link" href="solutions.html">SOLUTIONS</a></li>
					<li class="footer-nav__item"><a class="footer-nav__link" href="partners.html">PARTNERS</a></li>
					<li class="footer-nav__item"><a class="footer-nav__link" href="services.html">SERVICES</a></li>
					<li class="footer-nav__item"><a class="footer-nav__link" href="about.html">ABOUT</a></li>
				</ul>

				<ul class="footer-social">
					<li class="footer-social__item icon-one"><a class="footer-social__link" href="solutions.html"></a></li>
					<li class="footer-social__item icon-two"><a class="footer-social__link" href="partners.html"></a></li>
					<li class="footer-social__item icon-third"><a class="footer-social__link" href="services.html"></a></li>
					<li class="footer-social__item icon-four"><a class="footer-social__link" href="about.html"></a></li>
				</ul>
			</div>
		</div>
	</body>
</html>
