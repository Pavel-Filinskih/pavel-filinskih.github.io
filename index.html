<!DOCTYPE html>
<html class="no-js">
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Portfolio</title>
  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600' rel='stylesheet' type='text/css'>
  <link href="styles/main.css" rel="stylesheet" media="screen, print" type="text/css">
</head>
<body>
  <div id="container" style="left: 0px;">
    <section id="content">

      <header>
        <div id="preamble" class="home">
          <div class="preamble">
  	         <h1>Мои работы</h1>
          </div>
        </div>
      </header>

      <div id="our-work">
        <ul>
          <li><a href="http://pavfilin.ru/pink/"><img src="images/work/pink-preview.jpg" alt="Пинк"><div class="overlay"><summary><h2>PINK</h2><h3>Адаптивный мини-сайт, Sass + БЭМ</h3></summary></div></a></li>
          <li><a href="http://pavfilin.ru/mishka/"><img src="images/work/mishka-preview.jpg" alt="Мишка"><div class="overlay"><summary><h2>MISHKA</h2><h3>Адаптивный мини-сайт, Sass + БЭМ</h3></summary></div></a></li>
          <li><a href="http://pavfilin.ru/golden/"><img src="images/work/golden-preview.jpg" alt="Голден"><div class="overlay"><summary><h2>GOLDEN</h2><h3>Адаптивный мини-сайт, CSS + БЭМ</h3></summary></div></a></li>
          <li><a href="http://pavfilin.ru/nerds/"><img src="images/work/nerds-preview.jpg" alt="Нёрдс"><div class="overlay"><summary><h2>NERDS</h2><h3>Фиксированный мини-сайт, HTML + CSS + JS</h3></summary></div></a></li>
          <li><a href="http://pavfilin.ru/spirit/"><img src="images/work/nerds-preview.jpg" alt="Спирит"><div class="overlay"><summary><h2>SPIRIT</h2><h3>Фиксированный мини-сайт, HTML + Sass + БЭМ + JS</h3></summary></div></a></li>
        </ul>
      </div>
    </section>

    <footer id="msf">
      <div class="wrapper">
        <ul id="lets-be-social">
          <nav class="footer-social">
        	<a href="https://vk.com/pavelfilinskih" alt="vk" class="footer-social-item footer-social-item-vk" target="_blank" rel="nofollow">VK</a>
      	  </nav>
        </ul>
        <div id="legal">
        	<span>pavfilin<br>&copy; 2017</span>
        </div>
      </div>
    </footer>
  </div>
</body>
</html>
