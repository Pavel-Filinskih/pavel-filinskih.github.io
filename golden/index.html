<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>LP html css adaptiv</title>
    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,400i,700&amp;subset=cyrillic" rel="stylesheet">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/style.css">
  </head>
  <body>

    <header class="head">
      <div class="head__wrapper"> <!-- центровщик контента -->
        <a class="logo-link" href="#">
          <img class="logo-pict" src="img/logo.png" width="127" height="46" alt="Golden">
        </a>

        <menu class="navigation clearfix">
          <li class="navigation__item"><a class="navigation__link" href="#">начало</a></li>
          <li class="navigation__item"><a class="navigation__link" href="#">услуги</a></li>
          <li class="navigation__item"><a class="navigation__link" href="#">портфолио</a></li>
          <li class="navigation__item"><a class="navigation__link" href="#">контакты</a></li>
        </menu>

        <span class="head__welcom">Добро пожаловать в нашу студию!</span>
        <h1 class="head__title">Мы рады вам</h1>

        <a class="head-btn" href="#">Узнать больше</a>
      </div>
    </header>

    <main>
    <section class="services">
      <div class="services__wrapper">
        <h2 class="services__title">Услуги</h2>
        <span class="services__subtitle">Чем мы можем Вам помочь.</span>

        <article class="service-commerce">
          <img class="service-commerce__pict" src="img/basket-icon.png" alt="commerce">
          <h3 class="service-commerce__title">E-Commerce</h3>
          <p class="service-commerce__description">
            Большой опыт работы в E-commerce
            позволяет нам создавать реально
            эффективные продающие сайты
          </p>
        </article>

        <article class="service-web">
          <img class="service-web__pict" src="img/adaptiv-icon.png" alt="web">
          <h3 class="service-web__title">Responsive Web</h3>
          <p class="service-web__description">
            Всегда применяем только адаптивные
            технологии с использованием
            HTML5 и CSS3
          </p>
        </article>

        <article class="service-security">
          <img class="service-security__pict" src="img/lock-icon.png" alt="security">
          <h3 class="service-security__title">Web Security</h3>
          <p class="service-security__description">
            Являемся разработчиками ПО для усиления
            безопасности веб-сайтов, защитили более 1400
            проектов
          </p>
        </article>
      </div>
    </section>

    <section class="portfolio">
      <div class="portfolio__wrapper">
        <h2 class="portfolio__title">Наше портфолио</h2>
        <span class="portfolio__subtitle">Что мы сделали для любимых клиентов</span>

        <section class="ebony">
          <img class="ebony__pict" src="img/ebony_&_ivory.jpg" alt="картинка">
          <h4 class="ebony__title">Ebony & Ivory</h4>
          <span class="ebony__subtitle">Брендинг</span>
        </section>

        <section class="smart">
          <img class="smart__pict" src="img/smart_stationary.jpg" alt="картинка">
          <h4 class="smart__title">Smart Stationary</h4>
          <span class="smart__subtitle">Полиграфия</span>
        </section>

        <section class="сlever">
          <img class="сlever__pict" src="img/сlever_poster.jpg" alt="картинка">
          <h4 class="сlever__title">Clever Poster</h4>
          <span class="сlever__subtitle">Полиграфия</span>
        </section>

        <section class="vinyl">
          <img class="vinyl__pict" src="img/vinyl_record.jpg" alt="картинка">
          <h4 class="vinyl__title">Vinyl Record</h4>
          <span class="vinyl__subtitle">Обложка продукта</span>
        </section>

        <section class="treehouse">
          <img class="treehouse__pict" src="img/treehouse_template.jpg" alt="картинка">
          <h4 class="treehouse__title">Treehouse Template</h4>
          <span class="treehouse__subtitle">Веб дизайн</span>
        </section>

        <section class="burned">
          <img class="burned__pict" src="img/burned_logo.jpg" alt="картинка">
          <h4 class="burned__title">Burned Logo</h4>
          <span class="burned__subtitle">Брендинг</span>
        </section>
      </div>
    </section>

    <section class="about">
      <div class="about__wrapper">
        <h2 class="about__title">О нас</h2>
        <span class="about__subtitle">Кто мы такие</span>

        <div class="about__border">
        <article class="stage  stage--right">
          <img class="stage__pict  stage__pict--right" src="img/books.jpg" alt="картинка">
          <h4 class="stage__title">Июль 2010</h4>
          <span class="stage__subtitle">Наши скромные начинания</span>
          <p class="stage__description">
            Всё начиналось с небольшого офиса
            и двух человек в нём.
            С двух ноутбуков, большого
            энтузиазма и желании изменить мир.
          </p>
        </article>

        <article class="stage  stage--left">
          <img class="stage__pict  stage__pict--left" src="img/sea.jpg" alt="картинка">
          <h4 class="stage__title">январь 2011</h4>
          <span class="stage__subtitle">Первая премия за дизайн</span>
          <p class="stage__description">
            В январе 2011 мы получили первую
            премию в номинации “Лучший дизайн”
            по оценке Российского интернет форума.
            Мы радовались этому как дети, это
            действительно очень приятно!
          </p>
        </article>

        <article class="stage  stage--right">
          <img class="stage__pict  stage__pict--right" src="img/rockshow.jpg" alt="картинка">
          <h4 class="stage__title">декабрь 2012</h4>
          <span class="stage__subtitle">Первые неудачи</span>
          <p class="stage__description">
            Быть всегда успешными и везучими
            получается не всегда, и у нас как и у
            любой компании тоже были свои проблемы
            и неудачи. Но мы не отчаивались, а просто
            шли дальше и делали своё любимое дело.
          </p>
        </article>

        <article class="stage  stage--left">
          <img class="stage__pict  stage__pict--left" src="img/mountains.jpg" alt="картинка">
          <h4 class="stage__title">февраль 2014</h4>
          <span class="stage__subtitle">Наш триумф</span>
          <p class="stage__description">
            Во многом мы благодарны нашим любимым
            клиентам и партнёрам. Без них мы не смогли
            бы реализовать свой потенциал.
            В феврале 2014 в команде было уже 29 человек.
            И мы до сих пор растём!
          </p>
        </article>

        <div class="circle">
          <a class="circle__link" href="#">Другие наши истории ...</a>
        </div>
        </div>
      </div>
    </section>

    <section class="team">
      <div class="team__wrapper">
        <h2 class="team__title">Наша Команда</h2>
        <span class="team__subtitle">Наши ведущие специалисты</span>

        <article class="specialist">
          <img class="specialist__pict" src="img/anast.jpg" alt="картинка">
          <h4 class="specialist__name">Анастасия</h4>
          <span class="specialist__profession">Маркетолог</span>

          <div class="social-btn">
            <a class="social-btn__link tw" href="#"></a>
            <a class="social-btn__link fb" href="#"></a>
            <a class="social-btn__link pin" href="#"></a>
          </div>
        </article>

        <article class="specialist">
          <img class="specialist__pict" src="img/mark.jpg" alt="картинка">
          <h4 class="specialist__name">Марк</h4>
          <span class="specialist__profession">Программист</span>

          <div class="social-btn">
            <a class="social-btn__link tw" href="#"></a>
            <a class="social-btn__link fb" href="#"></a>
            <a class="social-btn__link gl" href="#"></a>
          </div>
        </article>

        <article class="specialist">
          <img class="specialist__pict" src="img/designer.jpg" alt="картинка">
          <h4 class="specialist__name">Виталий</h4>
          <span class="specialist__profession">Дизайнер</span>

          <div class="social-btn">
            <a class="social-btn__link fb" href="#"></a>
            <a class="social-btn__link pin" href="#"></a>
          </div>
        </article>

        <p class="team__slogan">Мы все любим свою работу и делаем её только на отлично. Иначе нас бы не было.</p>
      </div>
    </section>

    <div class="partners">
      <div class="partners__wrapper">
        <ul class="brand">
          <li class="brand__item"><a class="brand__link" href="#"><img class="brand__pict" src="img/envato.jpg" alt="картинка"></a></li>
          <li class="brand__item"><a class="brand__link" href="#"><img class="brand__pict" src="img/wordpress.jpg" alt="картинка"></a></li>
          <li class="brand__item"><a class="brand__link" href="#"><img class="brand__pict" src="img/aetuts.jpg" alt="картинка"></a></li>
          <li class="brand__item"><a class="brand__link" href="#"><img class="brand__pict" src="img/microlancer.jpg" alt="картинка"></a></li>
        </ul>
      </div>
    </div>

    <div class="map">
      <div class="map__wrapper">
        <script type="text/javascript" charset="utf-8" async src="https://api-maps.yandex.ru/services/constructor/1.0/js/?um=constructor%3A863d656952126e42b60ab8fa7fff462eb52369a20b5780425984adec06677eaa&amp;width=100%25&amp;height=400&amp;lang=ru_RU&amp;scroll=true"></script>
      </div>
    </div>

    <section class="contacts">
      <div class="contacts__wrapper">
        <h2 class="contacts__title">Напишите нам</h2>
        <span class="contacts__subtitle">Пишите, мы будем очень рады пообщаться с вами</span>

        <form class="contacts-form" method="post" action="form.php">
          <div class="contacts-form__cell  contacts-form__cell--left"> <!-- обертка блока -->
              <input class="contacts-form__input" type="text" name="user-name" placeholder="ВАШЕ ИМЯ *"/>
              <input class="contacts-form__input" type="email" name="email" placeholder="ВАШ E-MAIL *"/>
              <input class="contacts-form__input" type="tel" name="phone" placeholder="ТЕЛЕФОН"/>
          </div>
          <div class="contacts-form__cell  contacts-form__cell--right">
            <textarea class="user-message" name="message" placeholder="ВАШЕ СООБЩЕНИЕ *"></textarea>
          </div>
          <button class="contacts-form__btn" type="submit">Отправить</button>
        </form>
      </div>
    </section>
    </main>

    <footer class="footer">
      <div class="footer__wrapper">
        <p class="copyrighted">&copy; Copyright 2014 FreebiesXpress.com</p>
        <div class="social-btn social-btn--footer">
          <a class="social-btn__link footer-tw" href="#"></a>
          <a class="social-btn__link footer-fb" href="#"></a>
          <a class="social-btn__link footer-pin" href="#"></a>
          <a class="social-btn__link footer-gl" href="#"></a>
        </div>
      </div>
    </footer>

  </body>
</html>
